bin/build-schemas.ts

```diff
@@ -45,8 +51,9 @@ async function main() {
   }
 }
 
-process.on("unhandledRejection", (reason, promise) => {
+process.on("unhandledRejection", (reason: Error, promise) => {
   console.error("Unhandled Reject at:", promise, "reason:", reason);
```

Comment:
Should we omit `reason` here, since we're dumping the `reason.stack` below, or do we want both?

Reply:
`reason` is the value thrown, which is not always an error, (despite what the types say). Sometimes they don't have stacks, and either way the stack often doesn't have the details of what went wrong.

This is all one big hack anyways, since Node should be raising these messages itself.